static/templates/admin/ingestSourceMetadata.html

```diff
@@ -3,51 +3,51 @@
 		<a class="complete" id="ingest_source_choose" href="#">Choose materials</a>
 		<span class="bold"> &rarr; Assign metadata</span>
 	</div>
-	
+
 	<div class="errors hidden">
 		<div class="error_stack"></div>
 	</div>
-	
+
 	<div class="file_browse_list">
-	<table>
-	<%
-	_.each(selectedCandidates, function(candidate, index) { 
-		%>
-		<tr class="file_browse_entry">
-			<td class="file_browse_type_icon">
-				<% if (candidate.type == "BagIt") { %>
+		<table>
```

Comment:
Were all the changes in this file just formatting changes? I'm not actually seeing a checkbox for making something private here or when i run the branch